src/Chainweb/Cut.hs

```diff
@@ -497,21 +660,38 @@ applyJoin m = foldM
 -- the intersection. 'tryMonotonicCutExtension' is guaranteed to maintain that
 -- property.
 --
+-- Chains that aren't yet initialized are included in the join and later
+-- stripped from the result.
+--
+-- If you want to compute a join for cuts that include only a subset of all
```

Comment:
Suggest a `see comment on 'join' regarding subsets` here and below to save on comment maintenance

Reply:
I tent duplicate Haddock documentation, because many tools, like `ghci`s `:doc` command, or also pop-up windows in editors only show the documentation for the function itself and don't offer ways to easily follow cross-refs.

But I am not sure how important that is for these details...